### PR TITLE
updating box to stow (- name)

### PR DESCRIPTION
**Pull request by anujshah108**
changed everything over but the name of repo - need admin access for that - also fixed some search functions that were not working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,42 +80,42 @@ Go to your Metamask and switch to Ropsten Test Network
...
 
-To learn more about encryption patterns and how Linnia uses them, go [HERE](https://github.com/ConsenSys/linnia-resources/blob/master/KEYS.md)
+To learn more about encryption patterns and how Stow uses them, go [HERE](https://github.com/ConsenSys/stow-resources/blob/master/KEYS.md)
 
 
 
 ## Create User and Upload Data
 
-In order to create the keys, register a user and upload data to Linnia you can use our Linnia Faucet [HERE](https://consensys.github.io/linnia-faucet/)
+In order to create the keys, register a user and upload data to Stow you can use our Stow Faucet [HERE](https://consensys.github.io/stow-faucet/)
```
Comment (MaxBlaushild) [inline README.md]: https://faucet.ropsten.stow-protocol.com/

File: src/actions/AuthActions.js [hunk 2]
```diff
@@ -94,23 +94,23 @@ export const authenticate = () => async dispatch => {
...
   const code = await web3.eth.getCode(hubAddress);
   if (!code || code === '0x0' || code === '0x') {
-    console.error('Linnia is not configured correctly!');
-    return dispatch(authFailure(LINNIA_MISCONFIGURED));
+    console.error('Stow is not configured correctly!');
+    return dispatch(authFailure(STOW_MISCONFIGURED));
   }
 
-  const linnia = new Linnia(web3, { linniaContractUpgradeHubAddress: hubAddress });
+  const stow = new Stow(web3);
```
Comment (MaxBlaushild) [inline src/actions/AuthActions.js]: Should still be something like 

`const stow = new Stow(web3, { hubAddress })` so user can optionally pass in hubAddress when developing locally

File: src/config.js [hunk 3]
```diff
@@ -1,9 +1,9 @@
 const config = {
-  LINNIA_IPFS_HOST: 'ipfs.infura.io',
-  LINNIA_IPFS_PORT: '5001',
-  LINNIA_IPFS_PROTOCOL: 'https',
-  LINNIA_HUB_ADDRESS: '0x76305411d861ecc48153336be8b8c255e1aec823',
-  LINNIA_SEARCH_URI: 'https://linniaserver.com',
+  STOW_IPFS_HOST: 'ipfs.infura.io',
+  STOW_IPFS_PORT: '5001',
+  STOW_IPFS_PROTOCOL: 'https',
+  STOW_HUB_ADDRESS: '0xa8afd97368403ca160d4d8e02afa966100d97621',
```
Comment (MaxBlaushild) [inline src/config.js]: Should remove

File: src/actions/SearchActions.js [hunk 4]
```diff
@@ -11,30 +11,32 @@ const assignSearch = (search) => ({
...
 
-  let req = config.LINNIA_SEARCH_URI + "/records";
+  let req = config.STOW_SEARCH_URI + "/records";
 
   if (dataHash) {
     req = req + '/' + dataHash;
   } else if (owner) {
     if (property) {
-      req = req + '?owner=' + owner + '&property=' + property;
+      req = req + '/?owner=' + owner + '&property=' + property;
```
Comment (godfreyhobbs) [inline src/actions/SearchActions.js]: Is adding the slash necessary?

**Pull request merged**